common_benchmark_suite/openxla/benchmark/comparative_suite/pt/scripts/generate_model_artifacts.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Runs `generate_model_artifacts.py` on all registered PyTorch models and saves
+# artifacts into the directory `/tmp/pt_models_<torch-mlir-version>_<timestamp>`.
+#
+# Once complete. please upload the output directory to
+# `gs://iree-model-artifacts/pt`, preserving directory name.
+#
+# Usage:
+#     bash generate_saved_models.sh
+#
+# Requires python-3.11 and above and python-venv.
```

Comment:
I thought PyTorch would need to run on Python 3.10? Or it's just for PyTorch Inductor

Reply:
It's `torch-mlir` that needs 3.11 and above.